app/containers/MusicContainer/index.js

```diff
@@ -96,32 +92,32 @@ export function MusicContainer({
 
     return (
       (items.length !== 0 || loading) && (
-        <CustomCard>
+        <div>
+          {searchTerm && (
+            <div>
+              <T id="search_query" values={{ query: searchTerm }} />
+            </div>
+          )}
+          {totalCount !== 0 && (
+            <div>
+              <T id="matching_music" values={{ totalCount }} />
+            </div>
+          )}
           <Skeleton loading={loading} active>
-            {searchTerm && (
-              <div>
-                <T id="search_query" values={{ query: searchTerm }} />
-              </div>
-            )}
-            {totalCount !== 0 && (
-              <div>
-                <T id="matching_music" values={{ totalCount }} />
-              </div>
-            )}
-            {items.map((item, index) => (
-              <CustomCard key={index}>
-                <MusicContent>
-                  <Avatar src={item.artworkUrl100} shape="square" size={100} />
-                  <div style={{ paddingLeft: '0.6em' }}>
-                    <T id="music_title" values={{ title: item.trackName }} />
-                    <T id="artist_name" values={{ artistName: item.artistName }} />
-                    <T id="music_collection_name" values={{ collectionName: item.collectionName }} />
-                  </div>
-                </MusicContent>
-              </CustomCard>
-            ))}
+            <Row justify="center">
```

Comment:
this is creating new columns for every track

Reply:
Yes. It's looking fine